docs/getting-started.md

```diff
@@ -163,7 +163,7 @@ If you have already installed Node on your system, make sure it is Node 8.3 or n
 We recommend installing JDK using [Homebrew](http://brew.sh/). Run the following commands in a Terminal after installing Homebrew:
 
 ```sh
-brew cask install adoptopenjdk/openjdk/adoptopenjdk8
+brew cask install adoptopenjdk/openjdk/adoptopenjdk14
```

Comment:
The Open JDK 14 do not support Windows 7 and a bit older Linux distros. Are you sure that such big bump is required and recommended for the Android developers? 

* Open JDK 14: https://www.oracle.com/java/technologies/javase/products-doc-jdk14certconfig.html
* Open JDK 13: https://www.oracle.com/java/technologies/javase/products-doc-jdk13certconfig.html
* Open JDK 8: https://www.oracle.com/java/technologies/javase/products-doc-jdk8-jre8-certconfig.html

Reply:
You're right. I'll revert this change.